_posts/fr/2022-03-22-enquete-sur-le-crypto-gaming.md

```diff
@@ -87,25 +87,19 @@ La réponse ne me plaît pas : un jeton est créé à partir de rien, il n'a de
 
 Comme pour la crypto-monnaie classique, les tokens possèdent leur propre cours boursier.
 
-<div style="text-align: center;">
-    <img src="{{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/axie-slp.png" width="700px" alt="Cours de l'objet SLP du jeu Axie Infinity" style="display: block; margin: auto;"/>
-</div>
+![Cours de l'objet SLP du jeu Axie Infinity]({{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/axie-slp.png)
 
 *<center>Sur Axie, on peut gagner en combat des objets SLP qui permettent de faire de l'élevage.</center>*
 
 
 En fonction du cours, il sera ou non intéressant de les revendre, comme à la bourse.
 
-<div style="text-align: center;">
-    <img src="{{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/speculation-down.png" width="700px" alt="Starting game project" style="display: block; margin: auto;"/>
-</div>
+![Starting game project Up]({{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/speculation-down.png)
```

Comment:
width missing

Reply:
I did not put the width of 700 because the difference is hardly visible visually